sites/waterworld/server/templates/website-section/global-thought-leaders.marko

```diff
@@ -41,41 +49,105 @@ $ const { id, alias, name, pageNode } = data;
         </@section>
         <@section>
           <div class="row">
-            <div class="col-lg-8 mb-block">
-              <marko-web-query|{ nodes }|
-                name="website-scheduled-content"
-                params={ sectionId: id, limit: 2, contentTypes: ["Video"], queryFragment }
-              >
-                <default-theme-card-deck-flow cols=2 nodes=nodes>
-                  <@slot|{ node, index }|>
-                    <website-content-card-node
-                      image-width=340
-                      node=node
+            <marko-web-query|{ nodes }|
+              name="website-scheduled-content"
+              params={ sectionId: id, limit: 2, includeContentTypes: ["Video"], queryFragment }
+            >
+              <for|content| of=nodes>
+                <div class="col">
+                  <marko-web-node
+                    type="company-content"
+                    image-position="top"
+                    card=false
+                    full-height=true
+                  >
+                    <@image
+                      src=get(content, "company.primaryImage.src")
+                      alt=get(content, "company.primaryImage.alt")
+                      is-logo=get(content, "company.primaryImage.isLogo")
+                      fluid=true
+                      ar="16:9"
+                      width=520
+                      link={ href: get(content, "company.canonicalPath") }
                     />
-                  </@slot>
-                </default-theme-card-deck-flow>
-              </marko-web-query>
+                    <@body>
+                      <@header>
+                        <@left>Watch ${get(content, "company.name")}'s Video</@left>
+                      </@header>
+                      <@text><marko-web-content-embed-code block-name="node" obj=content /></@text>
+                    </@body>
+                  </marko-web-node>
+                </div>
+              </for>
+            </marko-web-query>
+          </div>
+          <div class="row">
+            <marko-web-query|{ nodes }|
+              name="website-scheduled-content"
+              params={ sectionId: id, limit: 2, includeContentTypes: ["Whitepaper"], queryFragment }
+            >
+              <for|content| of=nodes>
+                <div class="col">
+                  <marko-web-node
+                    type="whitepaper-content"
+                    image-position="top"
+                    card=false
+                    full-height=true
+                  >
+                    <@body>
+                      <@header>
+                        <@left>Download ${get(content, "company.name")}'s Whitepaper</@left>
+                      </@header>
+                      <@title>${content.name}</@title>
+                      <@text>$!{content.body}</@text>
```

Comment:
Use the built-in marko component for this: `<marko-web-content-body obj=content />`